ElasticBox/appliance-vsphere.md

```diff
@@ -50,11 +66,25 @@ In these steps, you connect the ElasticBox appliance to the vCenter datacenter n
 3. Now start the appliance. Right-click the appliance, click **Power > Power On**.
 
 ### Next Steps
+<<<<<<< HEAD
```

Comment:
Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.